src/index.html

```diff
@@ -7,12 +7,58 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac"
+        class="card__img"
+      />
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__review-amount">Reviews: 5</p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-title">Price:</p>
+        <p class="card__price-amount">$2,199</p>
+      </div>
+      <a
+        href="#"
+        class="card__buy"
+        data-qa="hover"
+      >
+        buy
+      </a>
```

Comment:
The text inside the link should be capitalized to match common e-commerce button styling conventions, consider using 'Buy' instead of 'buy'. Also, ensure that there is a visual indication (like underlining) when the link is hovered over, as per the checklist.